doc/sphinx/user-extensions/syntax-extensions.rst

```diff
@@ -687,13 +687,126 @@ side. E.g.:
    Notation "'apply_id' f a1 .. an" := (.. (f a1) .. an)
      (at level 10, f ident, a1, an at level 9).
 
+Custom entries
+~~~~~~~~~~~~~~
+
+.. cmd:: Declare Custom Entry @ident
+
+New term and pattern grammar entries, called *custom entries*, can be
```

Comment:
The content of the block should be indented at the level of `cmd` to be understood as the body of the command definition. See how I did it in my comment on your previous version.

Reply:
I knew it but forgot. Fixed.